admin.py

```diff
@@ -0,0 +1,183 @@
+def choose_the_option():
```

Comment:
If you only have one option, does it make sense to use this function? I'?m not sure